Lib/blackrenderer/backends/coregraphics.py

```diff
@@ -63,8 +63,9 @@ def _closePath(self):
 
 
 class CoreGraphicsCanvas(Canvas):
-    def __init__(self, context):
+    def __init__(self, context, colorSpace):
```

Comment:
colorSpace needs to have a default value of None, to be filled in with a colorSpace. Otherwise this is a breaking change, as it's a public class, and is used as such by FontGoggles.

Reply:
Firstly, I thought `CGBitmapContextGetColorSpace()` would work without adding an argument, but I realised that it doesn't work for `CGPDFContext`. Maybe I don't have to pass around the colorspace object but instead I should create an instance at `__init__`.